girder/api/rest.py

```diff
@@ -345,6 +368,10 @@ def wrapped(*args, **kwargs):
                     raise RestException(
                         'Invalid %s id (%s).' % (model.name, str(id)))
 
+                if self.requiredFlags and not self.force:
```

Comment:
Actually that is a weird case, I don't think that even makes sense. I'm more inclined to remove `and not self.force` from the line, which allows the user a little more power, albeit a very odd one I don't think anybody would want.